Unable to load JNA librar - kotlin 1.3.60

It seems that the update to kotlin 1.3.60 broke something.

Today I'm getting a lot of this kind of log errors:

```
WARN: Unable to load JNA library (OS: Linux 4.4.0-101-generic)
java.lang.NoClassDefFoundError: com/sun/jna/Native
	at org.jetbrains.kotlin.com.intellij.jna.JnaLoader.load(JnaLoader.java:16)
	at org.jetbrains.kotlin.com.intellij.jna.JnaLoader.isLoaded(JnaLoader.java:29)
	at org.jetbrains.kotlin.com.intellij.openapi.util.io.FileSystemUtil.getMediator(FileSystemUtil.java:54)
	at org.jetbrains.kotlin.com.intellij.openapi.util.io.FileSystemUtil.<clinit>(FileSystemUtil.java:46)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandler.setFileAttributes(ZipHandler.java:62)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandler$1.createAccessor(ZipHandler.java:44)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandler$1.createAccessor(ZipHandler.java:39)
	at org.jetbrains.kotlin.com.intellij.util.io.FileAccessorCache.createHandle(FileAccessorCache.java:61)
	at org.jetbrains.kotlin.com.intellij.util.io.FileAccessorCache.get(FileAccessorCache.java:54)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandler.getCachedZipFileHandle(ZipHandler.java:84)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandler.acquireZipHandle(ZipHandler.java:131)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ZipHandlerBase.createEntriesMap(ZipHandlerBase.java:44)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.ArchiveHandler.getEntriesMap(ArchiveHandler.java:183)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.jar.CoreJarHandler.<init>(CoreJarHandler.java:42)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.jar.CoreJarFileSystem.lambda$new$0(CoreJarFileSystem.java:33)
	at org.jetbrains.kotlin.com.intellij.util.containers.ConcurrentFactoryMap$2.create(ConcurrentFactoryMap.java:181)
	at org.jetbrains.kotlin.com.intellij.util.containers.ConcurrentFactoryMap.get(ConcurrentFactoryMap.java:46)
	at org.jetbrains.kotlin.com.intellij.openapi.vfs.impl.jar.CoreJarFileSystem.findFileByPath(CoreJarFileSystem.java:44)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment.findJarRoot(KotlinCoreEnvironment.kt:384)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment.contentRootToVirtualFile(KotlinCoreEnvironment.kt:363)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment.access$contentRootToVirtualFile(KotlinCoreEnvironment.kt:113)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$3.invoke(KotlinCoreEnvironment.kt:210)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$3.invoke(KotlinCoreEnvironment.kt:113)
	at org.jetbrains.kotlin.cli.jvm.compiler.ClasspathRootsResolver.convertClasspathRoots(ClasspathRootsResolver.kt:70)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment.<init>(KotlinCoreEnvironment.kt:217)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment.<init>(KotlinCoreEnvironment.kt:113)
	at org.jetbrains.kotlin.cli.jvm.compiler.KotlinCoreEnvironment$Companion.createForProduction(KotlinCoreEnvironment.kt:420)
	at io.gitlab.arturbosch.detekt.api.internal.KotlinEnvironmentUtilsKt.createKotlinCoreEnvironment(KotlinEnvironmentUtils.kt:38)
	at io.gitlab.arturbosch.detekt.core.ProcessingSettings$environment$2.invoke(ProcessingSettings.kt:88)
	at io.gitlab.arturbosch.detekt.core.ProcessingSettings$environment$2.invoke(ProcessingSettings.kt:22)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at io.gitlab.arturbosch.detekt.core.ProcessingSettings.getEnvironment(ProcessingSettings.kt)
	at io.gitlab.arturbosch.detekt.core.DetektFacade.<init>(DetektFacade.kt:29)
	at io.gitlab.arturbosch.detekt.core.DetektFacade$Companion.create(DetektFacade.kt:107)
	at io.gitlab.arturbosch.detekt.core.DetektFacade$Companion.create(DetektFacade.kt:91)
	at io.gitlab.arturbosch.detekt.cli.runners.Runner.execute(Runner.kt:29)
	at io.gitlab.arturbosch.detekt.cli.Main.main(Main.kt:16)
Caused by: java.lang.ClassNotFoundException: com.sun.jna.Native
	at java.net.URLClassLoader.findClass(URLClassLoader.java:381)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:335)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 37 more
```

And you can see them on Travis (linux and OSX) too.

I'm not sure if they are safe to ignore. And if so, cane we mute them? They are really noisy.

--- a/detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/internal/KotlinEnvironmentUtils.kt
+++ b/detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/internal/KotlinEnvironmentUtils.kt
@@ -2,6 +2,7 @@ package io.gitlab.arturbosch.detekt.api.internal
 
 import org.jetbrains.kotlin.cli.common.CLIConfigurationKeys
 import org.jetbrains.kotlin.cli.common.config.addKotlinSourceRoots
+import org.jetbrains.kotlin.cli.common.environment.setIdeaIoUseFallback
 import org.jetbrains.kotlin.cli.common.messages.MessageRenderer
 import org.jetbrains.kotlin.cli.common.messages.PrintingMessageCollector
 import org.jetbrains.kotlin.cli.jvm.compiler.EnvironmentConfigFiles
@@ -28,7 +29,7 @@ import java.nio.file.Path
  * This environment also allows to modify the resulting AST files.
  */
 fun createKotlinCoreEnvironment(configuration: CompilerConfiguration = CompilerConfiguration()): KotlinCoreEnvironment {
-    System.setProperty("idea.io.use.fallback", "true")
+    setIdeaIoUseFallback()
     configuration.put(
         CLIConfigurationKeys.MESSAGE_COLLECTOR_KEY,
         PrintingMessageCollector(System.err, MessageRenderer.PLAIN_FULL_PATHS, false)

--- a/detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/internal/KotlinEnvironmentUtils.kt
+++ b/detekt-api/src/main/kotlin/io/gitlab/arturbosch/detekt/api/internal/KotlinEnvironmentUtils.kt
@@ -29,6 +29,7 @@ import java.nio.file.Path
  * This environment also allows to modify the resulting AST files.
  */
 fun createKotlinCoreEnvironment(configuration: CompilerConfiguration = CompilerConfiguration()): KotlinCoreEnvironment {
+    // https://github.com/JetBrains/kotlin/commit/2568804eaa2c8f6b10b735777218c81af62919c1
     setIdeaIoUseFallback()
     configuration.put(
         CLIConfigurationKeys.MESSAGE_COLLECTOR_KEY,

--- a/detekt-gradle-plugin/build.gradle.kts
+++ b/detekt-gradle-plugin/build.gradle.kts
@@ -12,7 +12,7 @@ plugins {
     `maven-publish`
     id("com.gradle.plugin-publish") version "0.10.1"
     id("com.jfrog.bintray") version "1.8.4"
-    kotlin("jvm") version "1.3.50"
+    kotlin("jvm") version "1.3.60"
     id("org.jetbrains.dokka") version "0.10.0"
     id("com.github.ben-manes.versions") version "0.27.0"
     id("io.gitlab.arturbosch.detekt") version "1.2.0"

--- a/gradle.properties
+++ b/gradle.properties
@@ -17,7 +17,7 @@ buildScanVersion=3.0
 dokkaVersion=0.10.0
 gradleVersionsPluginVersion=0.27.0
 jacocoVersion=0.8.5
-kotlinVersion=1.3.50
+kotlinVersion=1.3.60
 shadowVersion=5.2.0
 sonarQubeVersion=2.8
 

